packages/web/src/components/usdc-purchase-details-modal/components/PurchaseModalContent.tsx

```diff
@@ -67,12 +104,16 @@ export const PurchaseModalContent = ({
       </ModalHeader>
       <ModalContent className={styles.content}>
         <Flex borderBottom='default' gap='l' w='100%' pb='xl'>
-          <DynamicTrackArtwork
-            id={purchaseDetails.contentId}
-            size={DynamicTrackArtworkSize.LARGE}
+          <DynamicImage
+            image={isTrack ? trackArtwork : albumArtwork}
+            wrapperClassName={styles.artworkContainer}
```

Comment:
nit: Idk if there's a reason to avoid another div but seems like you could replace this whole className with a harmony container (`Card` or `Paper` or something) wrapped around `DynamicImage`.

Reply:
took a gander, there's a wrapping `div` inside of `DynamicImage`, didn't feel like going down this sidequest at this point but agree keeping the css around is annoying